source/_posts/2023-03-01-php-core-roundup-10.md

```diff
@@ -0,0 +1,320 @@
+---
+title: 'PHP Core Roundup #10'
+layout: post
+tags:
+    - roundup
+author:
+  name: Ayesh Karunaratne
+  url: https://aye.sh
+published_at: 01 March 2023
+
+---
+
+Welcome back to the tenth in the PHP Core Roundup series. As February comes to end we have plenty of PHP developments to write about. February was a month full of new RFCs for upcoming PHP 8.3, merged changes, security releases for PHP 8.2, 8.1, and 8.0, as well as several discussions in the mailing list about improving PHP. 
+
+> The PHP Foundation currently supports six part-time PHP contributors who work on maintenance and new features for PHP. Maintenance is not limited to fixing bugs, but also includes work to reduce technical debt, making life easier for everyone working on PHP. The contributors funded by the PHP Foundation collaborate with other contributors on code, documentation, and discussions.
+
+<br>
+We publish the posts on our website, and you can subscribe to a newsletter; You don’t necessarily have to be a PHP Foundation backer to follow _PHP Core Roundup_.
+
+{% include "newsletter.html" %}
+
+## Call for PHP 8.3 Release Managers
```

Comment:
Today we published the deadline for applications and voting phase:
- Applications will be accepted until 31 March 12:00:00 UTC
- Elections (if needed) will start on 1 April and run until 16 April 12:00:00 UTC

You can add this if you like :)

Reply:
Awesome thank you @saundefined. I added these dates to the post.